README.md

```diff
@@ -9,88 +9,38 @@ Docker HUB url: https://hub.docker.com/r/dopplerrelay/doppler-relay-akamai-publi
-```
-
-In Linux shell:
-
-```bash
-export AKAMAI_CDN_HOSTNAME=example.cdn.com # CDN netstorage domain
-export AKAMAI_CDN_USERNAME=example         # Username from Akamai CDN
-export AKAMAI_CDN_PASSWORD=12345           # Password from Akamai CDN
-export AKAMAI_CDN_CPCODE=56789             # Cpcode provided by Akamai Netstorage, this is the root folder with write privileges inside the Akamai CDN
-```
-
-First Step, set the docker-compose.yml:
-
-The environment variables in the host machine are integrated in the docker image by the docker-compose.yml file. It is required by the cdn-uploader.js script.
+You need to define this environment variables. They are needed to login with the CDN ftp and cdn uploader script.
 
 ```bash
-version: '3.2'
-services:
-  app:
-    build:
-      context: .
-      dockerfile: editor.Dockerfile # Name of your dockerfile implementing doppler-docker-akamai-publish image
-    environment: 
-      - 'AKAMAI_CDN_HOSTNAME=${AKAMAI_CDN_HOSTNAME}'
-      - 'AKAMAI_CDN_USERNAME=${AKAMAI_CDN_USERNAME}'
-      - 'AKAMAI_CDN_PASSWORD=${AKAMAI_CDN_PASSWORD}'
-      - 'AKAMAI_CDN_CPCODE=${AKAMAI_CDN_CPCODE}'
+AKAMAI_CDN_HOSTNAME=example.cdn.com # CDN netstorage domain
+AKAMAI_CDN_USERNAME=example         # Username from Akamai CDN
+AKAMAI_CDN_PASSWORD=12345           # Password from Akamai CDN
+AKAMAI_CDN_CPCODE=56789             # Cpcode provided by Akamai Netstorage
+PROJECT_NAME=mseditor               # Change mseditor by your unique project name
+VERSION_NAME=v1.0.0-build1234       # Change this by your build identifier, this can be overrided dinamically using a bash script ex: docker run -e "VERSION_NAME=$1" this will receive $1 like a bash command line argument.
 ```
 
-Second Step, define your dockerfile:
-
-In order to implement the image you need to call it from a Dockerfile, you can multistage it with another one if you want.
+IMPORTANT: Environments variables defined with ENV in the dockerfile will be overrided by the host machine defined environment variables. This will be added in the README file.
 
-Example using multistage:
+## First Step, run it with bash script: ##
 
 ```bash
-FROM node:6.10.1 AS mseditor-env # You need to give an alias to previous executed images, in this example is: mseditor-env
-
-# Define your stuff...
-RUN mkdir /refactor
-RUN mkdir /refactor/builds
-WORKDIR /refactor
-RUN pwd
-COPY . .
-RUN npm install -g node-sass@4.8.3
-RUN npm install -g gulp
-RUN npm install
-RUN npm rebuild node-sass
-EXPOSE 3000
-RUN gulp default
-
-# Call a second stage to CDN upload using doppler-docker-akamai-publish image
-FROM dopplerrelay/doppler-relay-akamai-publish:latest
-WORKDIR .
-COPY --from=mseditor-env /refactor/builds /builds
-CMD ["node", "cdn-uploader.js", "mseditor", "builds"]
+docker run --rm \
+	-e AKAMAI_CDN_HOSTNAME \
+	-e AKAMAI_CDN_USERNAME \
+	-e AKAMAI_CDN_PASSWORD \
+	-e AKAMAI_CDN_CPCODE \
+	-e "PROJECT_NAME=mseditor" \
+	-e "VERSION_NAME=$1" \
```

Comment:
$1 has no value, I think that for this example you should hardcode the version

Reply:
Perfect, i will do it.